haiku-apps/calendar/calendar-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A native Calendar application for Haiku"
+DESCRIPTION="A native Calendar application for Haiku
```

Comment:
No need to repeat summary.

Reply:
DESCRIPTION=$SUMMARY , if you cannot find any better description...